src/Neo.Extensions/ByteExtensions.cs

```diff
@@ -17,7 +17,7 @@ namespace Neo.Extensions
 {
     public static class ByteExtensions
     {
-        private static readonly char[] s_hexChars = "0123456789abcdef".ToCharArray();
+        private const string s_hexChars = "0123456789abcdef";
```

Comment:
How can be const string faster than char string?

Reply:
If access to the char of a string is faster than access to the char array, the benchmark was wrong

Reply:
> If access to the char of a string is faster than access to the char array, the benchmark was wrong

UTF-8 string is slower, but this string just asc-ii, so it can be optimized.

Reply:
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/String.cs#L750-L753

Wrong benchmark, i'm sure

Reply:
> https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/String.cs#L750-L753
> 
> Wrong benchmark, i'm sure

Why this is slower, there are just two operations:  An `if` check and A pointer add, and it has an `Intrinsic ` attribute, so JIT can optimize it.

There are also a bound check(for `IndexOutOfRangeException`) and a pointer computation on byte array access.

Reply:
https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables#dotnet_jit-and-dotnet_gc

enable this on your computer and run dotnet in commandline to test JIT and GC outputs